README.md

```diff
@@ -113,7 +120,7 @@ $(PROJECT_DIR)/../libWebRTC/WebRTC.xcframework/ios-arm64_x86_64-simulator/WebRTC
 
```

Comment:
We have x64 builds also, we should consider them at the moment of adding the Search Paths